Pipfile

```diff
@@ -12,6 +12,8 @@ gunicorn = "*"
 django-heroku = "*"
 djangorestframework = "*"
 pillow = "*"
+djangorestframework-jwt = "*"
```

Comment:
Based on the update [here](https://github.com/jpadilla/django-rest-framework-jwt/issues/484), I think we should pivot to using [a newer, active library](https://github.com/davesque/django-rest-framework-simplejwt).